src/content/blog/astro-520.mdx

```diff
@@ -0,0 +1,183 @@
+
+As a result, the `@astrojs/tailwind` integration is now deprecated, but will continue to work for older versions of Tailwind. To upgrade to Tailwind 4, please uninstall the integration and either use the updated `astro add tailwind` command or follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro).
+
+Thanks to Eveeifyeve ([`@Eveeifyeve`](https://github.com/Eveeifyeve)) who did the first work on supporting Tailwind 4 while it was in alpha, even though this ended up using a different implementation.
+
+## Trailing slash redirects
+
+Whether or not a path should have a trailing slash is a common debate in web development. However everyone agrees that it should be consistent. Duplicate content is bad for SEO and confusing for users. Astro has long allowed you to configure whether routes should be matched with or without trailing slash, but previously this has meant returning a "not found" page for the incorrect path or leaving it up to the host to handle conflicts. Now, Astro will automatically redirect to the correct path on routes that are rendered on-demand, if you have configured the `trailingSlash` option in your `astro.config.mjs`.
+
+Now, it doesn't matter whether your visitor navigates to `/about/`, `/about`, or even `/about///`. In production, they'll always end up on the correct page. For GET requests, the redirect will be a 301 (permanent) redirect, and for all other request methods, it will be a 308 (permanent, and preserve the request method) redirect.
+
+During development, Astro will display an error page instead of redirecting to help you catch any misconfigurations and bad links.
+
+To enable trailing slash redirects, set the `trailingSlash` option in your `astro.config.mjs`:
+
+```js title="astro.config.mjs" ins={3}
+export default defineConfig({
+	adapter: node({ mode: 'standalone' }),
+	trailingSlash: 'never', // or 'always'
+});
+```
+
+For more information, see the [`trailingSlash` documentation](https://docs.astro.build/en/reference/configuration-reference/#trailingslash).
+
+## External redirects
+
+Astro now supports defining external redirects straight from the configuration. External redirects must start with `https` or `http`:
+
+```js
+import {defineConfig} from "astro/config"
+
+export default defineConfig({
+    redirects: {
+        "/about": "https://example.com/about",
+        "/news": {
+            status: 302,
+            destination: "https://example.com/news"
+        }
+    }
+})
+```
+
+
+## TOML frontmatter in Markdown
+
+Currently Astro supports YAML frontmatter in Markdown and MDX files. With this release, Astro now also supports TOML frontmatter in both. This is useful for adding existing content files with TOML frontmatter to your project from another framework such as Hugo, or you may just prefer TOML over YAML!
```

Comment:
```suggestion
Astro's excellent support for Markdown and MDX has always included the YAML frontmatter, whether in file exports or content collections. With this release, Astro now also supports TOML frontmatter in both. This is useful for adding existing content files with TOML frontmatter to your project from another framework such as Hugo, or you may just prefer TOML over YAML!
```
OK, this first sentence is kind of cringe, but I wanted to see if I could get somewhere with it. (You know and love YAML, YAML plays a big role in content collections, who else gives ya type-safe YAML?...)

BUT at a minimum, "currently" should be changed to "previously" or something else indicating past tense. If that's the only change, that's totally fine. 😄

Reply:
Nah that's fine, and as a father of a 13 and 11 year old I am an expert in being cringe.